src/MicroOcpp/Version.h

```diff
@@ -8,37 +8,49 @@
 /*
  * Version specification of MicroOcpp library (not related with the OCPP version)
  */
-#define MO_VERSION "1.2.0"
+#define MO_VERSION "2.0.0"
 
 /*
- * Enable OCPP 2.0.1 support. If enabled, library can be initialized with both v1.6 and v2.0.1. The choice
- * of the protocol is done dynamically during initialization
+ * OCPP version identifiers
  */
-#ifndef MO_ENABLE_V201
-#define MO_ENABLE_V201 0
-#endif
-
-#ifdef __cplusplus
-
-namespace MicroOcpp {
+#define MO_OCPP_V16  160 // OCPP 1.6
+#define MO_OCPP_V201 201 // OCPP 2.0.1
```

Comment:
We should use the IANA registered identifiers, the same used in the websocket subprotocol list. then we can reuse it there for switching versions.

https://www.iana.org/assignments/websocket/websocket.xml